admin/broadleaf-open-admin-platform/pom.xml

```diff
@@ -163,6 +163,8 @@
         <dependency>
             <groupId>org.hsqldb</groupId>
             <artifactId>hsqldb</artifactId>
+            <version>2.7.1</version>
```

Comment:
No need to define version on each of the POMs. It should come from parent POM. Not sure about the classifier so maybe test without it first.